reloc.ml

```diff
@@ -640,12 +640,19 @@ let patch_output filename =
    Indeed, we will create "__imp_X" (with a redirection to "X").
    Collect such cases in "imported".
 *)
-let needed imported defined unalias obj =
+let needed imported defined unalias unalternate obj =
```

Comment:
Apparently unalternate is a word, though not according to my browser's spell checker! I think `resolve_alternate` (and possibly therefore renaming `unalias` to `resolve_alias` but at least unalias does mean to resolve the alias!) 🙂

Reply:
Yep